docs/backends/index.rst

```diff
@@ -25,22 +25,31 @@ The library can inform you which backend is in affect, e.g.:
-| I/O                        | **no**      | **yes**   | **yes**  |   **yes**   |
-+----------------------------+-------------+-----------+----------+-------------+
-| I/O *(w/ metadata)*        | **no**      | **yes**   | **yes**  |   **no**    |
-+----------------------------+-------------+-----------+----------+-------------+
-| SGLs                       | **no**      | **no**    | **yes**  |   **no**    |
-+----------------------------+-------------+-----------+----------+-------------+
-| Async                      | **no**      | **yes**   | **yes**  |   **yes**   |
-+----------------------------+-------------+-----------+----------+-------------+
++----------------------------+-------------------------------------------------------------------+
+|                            | Backends                                                          |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| Async interfaces           | ``fbsd`` | ``linux`` | libvfn  | ``spdk`` | ``windows`` | ``mac`` |
++============================+==========+===========+=========+==========+=============+=========+
+| io_uring                   | no       | **yes**   | no      | no       | no          | no      |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| io_uring_cmd               | no       | **yes**   | no      | no       | no          | no      |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| libaio                     | no       | **yes**   | no      | no       | no          | no      |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| POSIX aio                  | **yes**  | **yes**   | no      | no       | no          | **yes** |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| NVMe Driver (vfio-pci)     | no       | no        | **yes** | **yes**  | no          | no      |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| NVMe Driver (uio-generic)  | no       | no        | no      | **yes**  | no          | no      |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| emu                        | **yes**  | **yes**   | **yes** | **yes**  | **yes**     | **yes** |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| thrpool                    | **yes**  | **yes**   | **yes** | **yes**  | **yes**     | **yes** |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
+| nil                        | **yes**  | **yes**   | **yes** | **yes**  | **yes**     | **yes** |
++----------------------------+----------+-----------+---------+----------+-------------+---------+
 
 
```

Comment:
maybe one for sync interfaces?

Reply:
Yeah, sync + mem + admin needs a similar functionality matrix plus some general usage description. I hold off on that until after this PR.